ckan/templates-midnight-blue/snippets/facet_list.html

```diff
@@ -37,56 +37,71 @@
-					{% if h.has_more_facets(name, search_facets) %}
-					    <a href="{{ h.remove_url_param('_%s_limit' % name, replace=0, extras=extras, alternative_url=alternative_url) }}" class="read-more">{{ _('Show More {facet_type}').format(facet_type=title) }}</a>
-					{% endif %}
-				    {% else %}
-					<a href="{{ h.remove_url_param('_%s_limit' % name, extras=extras, alternative_url=alternative_url) }}" class="read-more">{{ _('Show Only Popular {facet_type}').format(facet_type=title) }}</a>
-				    {% endif %}
-				</p>
-			    {% else %}
-				<p class="module-content empty">{{ _('There are no {facet_type} that match this search').format(facet_type=title) }}</p>
-			    {% endif %}
-			{% endwith %}
-		    {% endblock %}
-		</section>
-	    {% endblock %}
-	{% endif %}
-    {% endwith %}
-{% endblock %}
+	{% set hide_empty = hide_empty or false %}
+	{% with items = items or h.get_facet_items_dict(name, search_facets, exclude_active=true) %}
+		{% if items or not hide_empty %}
+			{% block facet_list_item %}
+				<section class="module module-narrow module-shallow accordion">
+					<div class="accordion-item">
+						{% block facet_list_heading %}
+							{% with items = items or h.get_facet_items_dict(name, search_facets) %}
+								{% set active_facet = h.currently_active_facet(name) %}
+								{% set collapsed = "collapsed" if default_collapsed and not active_facet else "" %}
+								<h2 class="accordion-header">
+									<button class="module-heading accordion-button {% if not items -%} collapsed {%- else -%} {{ collapsed }} {%- endif %}"
+										type="button"
+										data-bs-toggle="collapse"
+										data-bs-target="#{{ title }}"
+										{% if not items %}disabled{% endif %}>
+										<svg class="filter-svg" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
+											<path d="M2 4.6C2 4.03995 2 3.75992 2.10899 3.54601C2.20487 3.35785 2.35785 3.20487 2.54601 3.10899C2.75992 3 3.03995 3 3.6 3H20.4C20.9601 3 21.2401 3 21.454 3.10899C21.6422 3.20487 21.7951 3.35785 21.891 3.54601C22 3.75992 22 4.03995 22 4.6V5.26939C22 5.53819 22 5.67259 21.9672 5.79756C21.938 5.90831 21.8901 6.01323 21.8255 6.10776C21.7526 6.21443 21.651 6.30245 21.4479 6.4785L15.0521 12.0215C14.849 12.1975 14.7474 12.2856 14.6745 12.3922C14.6099 12.4868 14.562 12.5917 14.5328 12.7024C14.5 12.8274 14.5 12.9618 14.5 13.2306V18.4584C14.5 18.6539 14.5 18.7517 14.4685 18.8363C14.4406 18.911 14.3953 18.9779 14.3363 19.0315C14.2695 19.0922 14.1787 19.1285 13.9971 19.2012L10.5971 20.5612C10.2296 20.7082 10.0458 20.7817 9.89827 20.751C9.76927 20.7242 9.65605 20.6476 9.58325 20.5377C9.5 20.4122 9.5 20.2142 9.5 19.8184V13.2306C9.5 12.9618 9.5 12.8274 9.46715 12.7024C9.43805 12.5917 9.39014 12.4868 9.32551 12.3922C9.25258 12.2856 9.15102 12.1975 8.94789 12.0215L2.55211 6.4785C2.34898 6.30245 2.24742 6.21443 2.17449 6.10776C2.10986 6.01323 2.06195 5.90831 2.03285 5.79756C2 5.67259 2 5.53819 2 5.26939V4.6Z"/>
```

Comment:
did we need to embed the svg here, any possibility to use a svg->font load instead?